package.json

```diff
@@ -15,38 +15,27 @@
     "dev": "NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
     "watch": "NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
     "production": "NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "lint": "eslint resources/assets/js --ext .js,.vue --fix && exit 0",
-    "lint-css": "stylelint \"resources/assets/css/**/*.css\" --fix"
+    "prettier": "prettier --write \"**/*.{css,js,vue}\""
```

Comment:
Is it an idea to keep `yarn lint` as commando? (Like we also use `yarn dev` instead of `yarn webpack` for example)

Reply:
Makes sense! I'll rename it.

Reply:
Actually, I think `format` is a better generic command name, since that's what prettier does. Linting implies checking for non-style related rules too.

Reply:
Other ecosystems (Go, Elixir, Rust) all have a dedicated `format` command too.

Reply:
👍